evennia/accounts/accounts.py

```diff
@@ -189,6 +191,41 @@ def nicks(self):
     @lazy_property
     def sessions(self):
         return AccountSessionHandler(self)
+        
+    def get_absolute_url(self):
```

Comment:
I think these should be marked somehow to note that they are web-specific methods. Maybe prepend all such methods with `web_` or somesuch? They make no real sense for people coming at them from the pure Python side.

Reply:
Fair point, but get_absolute_url is a Django construct and the Admin pane looks for this function when generating reverse links to go back to the object detail page from the Admin page. The rest of the functions I felt made more sense to follow this convention for as well. Your call...

Reply:
I still think marking web-specific methods with something like  `web_`, yes. In the interest of consistency, this method should then also be renamed such - it's easy enough to add an alias just below (with an explaining comment) aliasing it to the name Django expects.